gcloud/streaming/http_wrapper.py

```diff
@@ -220,13 +236,13 @@ def RebuildHttpConnections(http):
                 del http.connections[conn_key]
 
 
-def HandleExceptionsAndRebuildHttpConnections(retry_args):
+def handle_http_exceptions(retry_args):
```

Comment:
You took away part of the name but the functionality is the same. Thoughts?

PS I am slightly scared that an exception means something needs to be rebuilt.